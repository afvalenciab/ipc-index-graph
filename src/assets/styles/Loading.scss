.container--loading {
  grid-row: 2 / 4;
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100%;

  h1 {
    font-size: 30px;
    letter-spacing: 3px;
    animation-name: loading;
    animation-duration: 1s;
    animation-iteration-count: infinite;
    animation-direction: alternate;
  }
}

@keyframes loading {
  0%{
    opacity: 1;
    transform: scale(1);
  }
  100%{
    opacity: .3;
    transform: scale(.6);
  }
}