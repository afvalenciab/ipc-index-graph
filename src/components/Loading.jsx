import React from 'react';
import '../assets/styles/Loading.scss';

const Loading = () => {
  return (
    <section className='container--loading'>
      <h1>Loading...</h1>
    </section>
  );
};

export default Loading;
