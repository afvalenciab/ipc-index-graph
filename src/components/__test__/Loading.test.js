import React from 'react';
import { create } from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import { configure } from 'enzyme';
import Loading from '../Loading';

configure({ adapter: new Adapter() });

describe('Loading testing', () => {
  test('Match Snapshot', () => {
    const loading = create(<Loading />);
    expect(loading.toJSON()).toMatchSnapshot();
  });
});