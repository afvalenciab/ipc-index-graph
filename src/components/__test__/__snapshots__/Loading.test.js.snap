// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading testing Match Snapshot 1`] = `
<section
  className="container--loading"
>
  <h1>
    Loading...
  </h1>
</section>
`;
